app/experimenter/nimbus-ui/src/components/PageRequestReview/FeedbackRejected/index.test.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import FeedbackRejected from ".";
+
+describe("FeedbackRejected", () => {
```

Comment:
Also, I know I could have just stuck an istabul ignore in here, but this seemed simple enough to test for now and will be easy to replace with the "real" (mock) data later.